package.json

```diff
@@ -84,7 +84,7 @@
     "zx": "^4.2.0"
   },
   "peerDependencies": {
-    "@stripe/connect-js": ">=3.3.21-preview-1",
+    "@stripe/connect-js": ">=3.3.21-preview-2",
```

Comment:
This version does not exist. I will fix it in a separate PR.

Reply:
thanks for catching!